docs/README.md

```diff
@@ -37,6 +37,7 @@ description: Starship is the minimal, blazing fast, and extremely customizable p
    ```sh
    curl -fsSL https://starship.rs/install.sh | bash
    ```
+   To update starship itself, rerunnning the script above will replace the current version, keeping configs untouched.
```

Comment:
```suggestion
   To update the Starship itself, rerun the above script. It will replace the current version without touching Starship's configuration.
```